cpp/tests/test_odeseir.cpp

```diff
@@ -52,85 +64,33 @@ class ModelTestOdeSeir : public testing::Test
 
         total_population = 1061000;
 
-        model.populations[{mio::Index<mio::oseir::InfectionState>(mio::oseir::InfectionState::Exposed)}]   = 10000;
-        model.populations[{mio::Index<mio::oseir::InfectionState>(mio::oseir::InfectionState::Infected)}]  = 1000;
-        model.populations[{mio::Index<mio::oseir::InfectionState>(mio::oseir::InfectionState::Recovered)}] = 1000;
-        model.populations[{mio::Index<mio::oseir::InfectionState>(mio::oseir::InfectionState::Susceptible)}] =
-            total_population -
-            this->model.populations[{mio::Index<mio::oseir::InfectionState>(mio::oseir::InfectionState::Exposed)}] -
-            this->model.populations[{mio::Index<mio::oseir::InfectionState>(mio::oseir::InfectionState::Infected)}] -
-            this->model.populations[{mio::Index<mio::oseir::InfectionState>(mio::oseir::InfectionState::Recovered)}];
-        // suscetible now set with every other update
-        // model.nb_sus_t0   = model.nb_total_t0 - model.nb_exp_t0 - model.nb_inf_t0 - model.nb_rec_t0;
+        model.populations[{mio::AgeGroup(0), mio::oseir::InfectionState::Exposed}]   = 10000;
+        model.populations[{mio::AgeGroup(0), mio::oseir::InfectionState::Infected}]  = 1000;
+        model.populations[{mio::AgeGroup(0), mio::oseir::InfectionState::Recovered}] = 1000;
+        model.populations[{mio::AgeGroup(0), mio::oseir::InfectionState::Susceptible}] =
+            total_population - model.populations[{mio::AgeGroup(0), mio::oseir::InfectionState::Exposed}] -
+            model.populations[{mio::AgeGroup(0), mio::oseir::InfectionState::Infected}] -
+            model.populations[{mio::AgeGroup(0), mio::oseir::InfectionState::Recovered}];
         model.parameters.set<mio::oseir::TransmissionProbabilityOnContact>(1.0);
         model.parameters.set<mio::oseir::TimeExposed>(5.2);
         model.parameters.set<mio::oseir::TimeInfected>(2);
 
-        model.parameters.get<mio::oseir::ContactPatterns>().get_baseline()(0, 0) = 2.7;
-        model.parameters.get<mio::oseir::ContactPatterns>().add_damping(0.6, mio::SimulationTime(12.5));
+        mio::ContactMatrixGroup& contact_matrix =
+            model.parameters.get<mio::oseir::ContactPatterns>().get_cont_freq_mat();
+        contact_matrix[0].get_baseline().setConstant(2.7);
+        contact_matrix[0].add_damping(0.6, mio::SimulationTime(12.5));
     }
-
-public:
-    double t0;
-    double tmax;
-    double dt;
-    double total_population;
-    mio::oseir::Model model;
 };
 
-TEST_F(ModelTestOdeSeir, compareWithPreviousRun)
```

Comment:
Do we want to delete these 'compareWithPreviousRun' tests in general? Because as far as I know, we have these tests in the other models. If we delete it here, we should delete it for the other models as well.